components/waitwhile/actions/update-customer-note-entry/update-customer-note-entry.ts

```diff
@@ -0,0 +1,54 @@
+import waitwhile from "../../app/waitwhile.app";
+import { defineAction } from "@pipedream/types";
+
+export default defineAction({
+  name: "Update Customer Note Entry",
+  version: "0.0.1",
+  key: "waitwhile-update-customer-note-entry",
+  description: "Update a customer note entry",
+  props: {
+    waitwhile,
+    customerId: {
+      propDefinition: [
+        waitwhile,
+        "customerId",
+       
+      ],
+    },
+    noteId: {
+      propDefinition: [
+        waitwhile,
+        "customerNoteId",
+        (c) => ({
+          customerId: c.customerId,
+        }),
+      ],
+    },
+    content: {
+      propDefinition: [
+        waitwhile,
+        "content"
+      ],
+    }
+
+  },
+  type: "action",
+  methods: {},
+  async run({ $ }) {
+    const params = {
+      customerId: this.customerId,
+      noteId: this.noteId,
+    };
+
+    try {
+      const data = await this.waitwhile.updateCustomerNoteEntry(params);
+    $.export("summary", "Successfully updated customer note entry");
+    return data;
```

Comment:
Looks like an indentation error
you should use a auto format tool to avoid this kind of issue